docs/releases/2.8.0.rst

```diff
@@ -15,9 +15,11 @@ If you want to try it, check one of the following:
 Changes in Requirements
 =======================
 
-- Django>=1.9.10,<1.10
+- Django>=1.9.11,<1.10
 - zlib1g-dev required for lxml
 - MySQL support uses mysqlclient instead of MySQLdb
+- django-transaction-hooks is no longer used
```

Comment:
Yes, this [arrived in 2.7.1](https://github.com/translate/pootle/commit/cd2fc9fbc180fd3dd62805fa03ca4a4ebd9de521).